lib/challenge.js

```diff
@@ -48,7 +62,19 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet {
+
+name: "John Doe",
+species: "Dob",
+breed: "Chia",
+noise: [
```

Comment:
This property contains a reference to an array that contains one object literal. The actual noise strings are stored in the object literal's properties. The array seems to serve no purpose here.